### PR TITLE
Added key prefix to key translations that are common words

### LINKED ISSUE #2180: Some key names need to be prefixed with KEY_
Keys that are common words like: end, delete, home, break / pause, help, refresh should have the translation keys be "KEY_END" to do this I think changes are needed in the keynames file with the switch case having specific mapping for them with an explicit call to the TranslationServer.Translate. That way the key names are no longer needed in the huge list of the Translate calls.

Edit some more key names:
- Forward
- Tab
- Enter
- Insert
- Pause
- Clear
- Home
- End
- Left
- Up
- Right
- Down
- Menu
- Help
- Back
- Forward
- Stop
- Refresh
- Search
- Standby
- Open URL
- Mail

### PR DESCRIPTION
**Pull request by hhyyrylainen**
closes #2180

### CODE REVIEW

File: src/engine/input/key_mapping/KeyNames.cs [hunk 1]
```diff
@@ -121,6 +121,35 @@ public static string Translate(uint keyCode)
...
+            KeyList.Down => TranslationServer.Translate("KEY_DOWN"),
+            KeyList.Menu => TranslationServer.Translate("KEY_MENU"),
+            KeyList.Help => TranslationServer.Translate("KEY_HELP"),
+            KeyList.Back => TranslationServer.Translate("KEY_BACK"),
+            KeyList.Stop => TranslationServer.Translate("KEY_STOP"),
+            KeyList.Refresh => TranslationServer.Translate("KEY_REFRESH"),
+            KeyList.Search => TranslationServer.Translate("KEY_SEARCH"),
+            KeyList.Standby => TranslationServer.Translate("KEY_STANDBY"),
+            KeyList.Openurl => TranslationServer.Translate("KEY_OPENURL"),
+
```
Comment (FuJa0815) [inline src/engine/input/key_mapping/KeyNames.cs:147]: huh, why is this empty line here?

Comment (hhyyrylainen) [reply src/engine/input/key_mapping/KeyNames.cs:147]: Forgot to delete it when I was merging the lists I edited together...

Comment (hhyyrylainen) [reply src/engine/input/key_mapping/KeyNames.cs]: Removed.

**Review by FuJa0815 — APPROVED** [review]
Should be fine

**Pull request merged**